mne/preprocessing/ica.py

```diff
@@ -2582,6 +2582,7 @@ def plot_sources(
         title=None,
         show=True,
         block=False,
+        psd_args=None,
```

Comment:
This should be placed after the `*` otherwise it's technically a backward compat break